datadog/resource_datadog_integration_azure.go

```diff
@@ -145,6 +152,7 @@ func buildDatadogAzureIntegrationDefinition(terraformDefinition *schema.Resource
 	// Optional params
 	hostFilters := terraformDefinition.Get("host_filters")
 	datadogDefinition.SetHostFilters(hostFilters.(string))
+	datadogDefinition.SetAutomute(terraformDefinition.Get("automute").(bool))
```

Comment:
```suggestion
	automute := terraformDefinition.Get("automute")
	datadogDefinition.SetAutomute(automute.(bool))
```
just a small suggestion to keep the same style as hostFilters..., otherwise LGTM.